config/default.js

```diff
@@ -136,4 +136,10 @@ module.exports = {
       missing: 'Workflow \'%workflow\' for Content Type \'%type\' not found',
     },
   },
+  storage: {
+    type: 'fs',
```

Comment:
This uses any [Vinyl Adapter](https://www.npmjs.com/package/vinyl#what-is-a-vinyl-adapter) that your heart can desire. There's no possible way we can document all the possibilities